### PR TITLE
Use correct function names in documentation

### PR DESCRIPTION
**Pull request by guggero**
While reading the documentation I noticed some function names have changed since this text was written. I hope I picked the right ones.

Some line breaks had to be changed too as to respect the maximum line length.

### CODE REVIEW

**Comment by apoelstra** [comment]
Thank you!

I think with this file we probably need to mostly rewrite it; I added it to the repo to help with review/further development because it was the most complete description of the algebra that we had laying around. This PR is still very WIP.

cc @jonasnick

**Comment by guggero** [comment]
I'm aware this PR is still WIP. But I was reading through the description and the code and noticed some things (see #10).
I thought it might be helpful for others who have an early look at your work and want to understand it or get it running.

File: src/modules/thresholdsig/threshold.md [hunk 1]
```diff
@@ -57,26 +57,27 @@ Key Generation works as follows.
 2. Each signer uses `secp256k1_musig_pubkey_combine` to compute the combined
    public key `P`. This function also outputs a list `Y_i` of tweaked public
    keys, which will be needed to validate partial signatures.
-3. Each signer modifies her secret key using `secp256k1_musig_tweak_secret_key`
-   to produce a secret key `y_i` compatible with her public key `Y_i`.
+3. Each signer modifies her secret key by multiplying it with the coefficient
```
Comment (jonasnick) [inline src/modules/thresholdsig/threshold.md]: I don't think this multiplication happens. I also left a comment about that in the original PR.

File: src/modules/thresholdsig/threshold.md [hunk 2]
```diff
@@ -106,39 +107,40 @@ verify_signature
 To produce a signature, each signer `i` acts as follows.
 
 1. Produces a nonce pair `(k_i, R_i)` and a commitment (hash) `C_i` to `R_i`.
-   This is done with `secp256k1_musig_multisig_generate_nonce`. Sends the
+   This is done with `secp256k1_musig_session_get_public_nonce`. Sends the
```
Comment (jonasnick) [inline src/modules/thresholdsig/threshold.md]: nonce and commitment are created with `secp256k1_musig_session_initialize` and the nonce can be obtained with `secp256k1_musig_session_get_public_nonce`.

File: src/modules/thresholdsig/threshold.md [hunk 3]
```diff
@@ -106,39 +107,40 @@ verify_signature
...
+   signatures.
 
 Signing Procedure Flowchart:
 ```
 Me:                           Others:
 
-multisig_generate_nonce C ->
-                           <- multisig_generate_nonce C
-signer_data_initialize
+musig_generate_nonce C    ->
```
Comment (jonasnick) [inline src/modules/thresholdsig/threshold.md]: There's no `musig_generate_nonce`. This is now done in `session_initialize`. How about swapping `musig_generate_nonce` with `session_initialize` and replacing `musig_generate_nonce` with `musig_get_public_nonce`?

**Review by jonasnick — CHANGES_REQUESTED** [review]
Thanks for your help! I left a few minor comments.

**Comment by guggero** [comment]
@jonasnick Thank you for your input!
I've updated my changes accordingly.

**Comment by jonasnick** [comment]
ACK 0a734caea84fc6c2f16d9c37562b84198312d09c

**Comment by jonasnick** [comment]
This is ready to merge imho

**Comment by apoelstra** [comment]
Sure

**Pull request merged**